crmsh/ui_cluster.py

```diff
@@ -91,78 +138,94 @@ def __init__(self):
         self._inventory_target = None
 
     @command.skill_level('administrator')
-    def do_start(self, context):
+    def do_start(self, context, *args):
         '''
-        Starts the cluster services on this node
+        Starts the cluster services on all nodes or specific node(s)
         '''
-        try:
-            if utils.service_is_active("pacemaker.service"):
-                logger.info("Cluster services already started")
-                return
-            bootstrap.start_pacemaker()
-            if utils.is_qdevice_configured():
-                utils.start_service("corosync-qdevice")
-            logger.info("Cluster services started")
-        except IOError as err:
-            context.fatal_error(str(err))
-
-        # TODO: optionally start services on all nodes or specific node
+        node_list = parse_option_for_nodes(context, *args)
+        for node in node_list[:]:
+            if utils.service_is_active("pacemaker.service", remote_addr=node):
+                logger.info("Cluster services already started on {}".format(node))
+                node_list.remove(node)
+        if not node_list:
+            return
+
+        bootstrap.start_pacemaker(node_list)
+        if utils.is_qdevice_configured():
+            utils.start_service("corosync-qdevice", node_list=node_list)
```

Comment:
`bootstrap.start_pacemaker` here is to start pacemaker.service, according the dependency, corosync.service will be started firstly, then start qdevice does make sense; Otherwise, start qdevice without corosync started will be failed:)

Reply:
Given that corosync-qdevice.service has these defined as well:
```
Wants=corosync.service
After=corosync.service
```
I'd expect pure start of qdevice.service will resolve the dependency as well.

Reply:
I can see a little challenge for the current code to check and start pacemaker and qdevice altogether before report the correct information. This may need more lines of code than expect.

Reply:
I can get the point to move corosync-qdevice.service start operation before pacemaker. There is no harm to have multiple systemctl start, in theory.

Reply:
Done